lib/amorail/entity/persistance.rb

```diff
@@ -40,5 +40,21 @@ def reload
     def extract_data_add(response)
       response.fetch('add').first
```

Comment:
I made some requests with empty required fields and amoCRM respond with 400, so I got `Amorail::AmoBadRequestError` exception.
I will try some more, but this method can be called only if 200 or 204 was returned.
Unlikely there can be errors in such cases.